### PR TITLE
car docs: fix docs diff on github

### PR DESCRIPTION
**Pull request by maxime-desroches**

### CODE REVIEW

**Comment by sshane** [comment]
think it might be worth it to just write a simple unit test that uses dump and print on the same commit to make sure there's no diff, that would explicitly test this

File: .github/workflows/selfdrive_tests.yaml [hunk 1]
```diff
@@ -370,7 +370,7 @@ jobs:
         id: save_diff
         run: |
           ${{ env.RUN }} "scons -j$(nproc)"
-          output=$(${{ env.RUN }} "python selfdrive/debug/print_docs_diff.py --path /tmp/openpilot_cache/base_car_info") || true
+          output=$(${{ env.RUN }} "python selfdrive/debug/print_docs_diff.py --github --path /tmp/openpilot_cache/base_car_info")
```
Comment (sshane) [inline .github/workflows/selfdrive_tests.yaml:373]: this can be separate from this PR

Comment (sshane) [inline .github/workflows/selfdrive_tests.yaml:373]:
```suggestion
          output=$(${{ env.RUN }} "python selfdrive/debug/print_docs_diff.py --github --path /tmp/openpilot_cache/base_car_info") || true
```

File: selfdrive/debug/print_docs_diff.py [hunk 2]
```diff
@@ -107,11 +109,15 @@ def print_car_info_diff(path):
           markdown_builder.append(COLUMN_HEADER)
         markdown_builder.extend(changes[category])
 
-    print("\n".join(markdown_builder))
+    diff = "\n".join(markdown_builder)
+    if github:
+      diff = diff[:MAX_GITHUB_COMMENT_SIZE] + '\n...\nTo see the full diff, run ``` python selfdrive/debug/print_docs_diff.py ```'
```
Comment (sshane) [inline selfdrive/debug/print_docs_diff.py:114]: does `MAX_GITHUB_COMMENT_SIZE` take into account the extra message added?

**Review by sshane — APPROVED** [review]

**Comment by sshane** [comment]
@maxime-desroches as long as it fails when failing to post the comment, we can close this for now

File: selfdrive/debug/print_docs_diff.py (hunk 2 above)
Comment (maxime-desroches) [reply selfdrive/debug/print_docs_diff.py:114]: yes it does

**Comment by sshane** [comment]
Probably going to add more complexity to format it properly, want to reduce complexity so going to close for now

**Pull request closed without merging**